data/categories.txt

```diff
@@ -6485,6 +6485,56 @@ sw:Malazi
 fa:پناهگاه
 mr:निवारा
 
+amenity-shelter-basic_hut|@category_hotel
+en:3Bivouac Hut
+ar:كوخ إقامة مؤقتة
+be:хаціна-бівак
+bg:хижа за бивак
+ca:cabana bivac
+cs:bivakovací chata
+da:Bivuakhytte
+de:4Biwakschachtel
+el:καταφύγιο bivouac
+es:cabaña de vivac
+es-MX:cabaña vivac
+et:bivouac-küünla
+eu:bivouac borda
+fa:کلبه بیواک
+fi:bivouac-mökki
+fr:cabane de bivouac
+he:בקתת ביבוק
+hi:द्विवार्षिक झोपड़ी
+hu:bivak kunyhó
+id:pondok bivak
+it:capanna di bivacco
+ja:ビバーク小屋
+ko:비박 오두막
+mr:बिव्होक झोपडी
+nb:bivuakkhytte
+nl:bivakhut
+pl:chata biwakowa
+pt:cabana de bivaque
+ro:baracă de bivuac
+ru:Бивуачная хижина
+sk:bivaková chata
+sv:bivack hydda
+sw:kibanda cha bivouac
+th:กระท่อมพักแรม
+tr:bivouac kulübe
+uk:курінь для бівуаку
+vi:túp lều bivouac
+zh-Hans:行军床
+zh-Hant:露營小屋
+
+amenity-shelter-lean_to|@category_hotel
+en:3Lean-to Shelter
+be:Бівальны навес
+bg:Подслон за бивак
+de:Lean-to Wetterschutz
+he:מחסה נשען
```

Comment:
```suggestion
he:מחסה ללינה|מחסה|מחסה פתוח
```
meaning: shelter for sleeping | shelter | open shelter

they all start with the word "shelter" (מחסה), does that make the middle one redundant or does it still offer something? (I've never played with this part of Organic Maps)

Reply:
technically its redundant, but if removing it makes the phrase weird, then its OK to keep it